### ISSUE TITLE
Discriminators

### ISSUE THREAD

**Issue by kurapan** [saic-mdal/HiDT#13]
Hello!
I would like to ask about the discriminators used for training of the model. The repository includes only inference time code and the discriminators aren't included so I wanted to implement the discriminators for training based on the paper, but there is hardly any information about the architecture of the discriminators given in the paper. You say that you use a conditional and and unconditional discriminator and they are trained with LSGAN loss, but what are the details of the architecture of the discriminators? It would really help if you could provide the details so that it is possible to reimplement the training code and reproduce the results.
